goWrapper/git_push.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
what is the need of this file?

Reply:
It was autogenerated, didn't created any hence removed it.